startup.py

```diff
@@ -0,0 +1,3 @@
+import modmesh.view as mv
```

Comment:
It is not necessary to use a file to run code like this.  A one-liner `python3 -c 'import modmesh.view as v ; v.launch()'` would do it.

Reply:
But I'm not sure which file is appropriate to add this code. Is there anything wrong to add the `startup.py` in modmesh?

Reply:
You do not need to add the code in a file.  That is a one-liner you type in command line.